### PR TITLE
chore(RHTAPWATCH-1159): add pipelinerun name to message sent to SNS

### PR DESCRIPTION
**Pull request by avi-biton**
add the pipelinerun name to the results extracted
from the pipelinerun

### CODE REVIEW

File: internal/controller/pipelinerun_helper.go [hunk 1]
```diff
@@ -49,9 +49,16 @@ func RemoveFinalizerFromPipelineRun(ctx context.Context, pipelineRun *tektonv1.P
 }
 
 // GetResultsFromPipelineRun extracts results from pipelinerun
+// And add the pipelinerunName
 // Return error if failed to extract results
 func GetResultsFromPipelineRun(pipelineRun *tektonv1.PipelineRun) ([]byte, error) {
-	results, err := json.Marshal(pipelineRun.Status.Results)
+	addPipelineRunName := tektonv1.PipelineRunResult{
```
Comment (yftacherzog) [inline internal/controller/pipelinerun_helper.go:55]:
```suggestion
	namedResults := tektonv1.PipelineRunResult{
```

File: internal/controller/pipelinerun_helper.go [hunk 2]
```diff
@@ -49,9 +49,16 @@ func RemoveFinalizerFromPipelineRun(ctx context.Context, pipelineRun *tektonv1.P
...
+		Name:  "PIPELINERUN_NAME",
+		Value: *tektonv1.NewStructuredValues(pipelineRun.Name),
+	}
+	fatchResults := pipelineRun.Status.Results
```
Comment (yftacherzog) [inline internal/controller/pipelinerun_helper.go:59]:
```suggestion
	fetchedResults := pipelineRun.Status.Results
```

**Review by yftacherzog — COMMENTED** [review]
I think the names could be improved. Added some suggestions.
Also, how come this does not affect the tests we have?

**Comment by avi-biton** [comment]
> how come this does not affect the tests we have?

Why should it?
The integration tests uses a mock to notify and we did not add unit tests to these functions

File: internal/controller/pipelinerun_helper.go (hunk 1 above)
Comment (avi-biton) [reply internal/controller/pipelinerun_helper.go]: Renamed

File: internal/controller/pipelinerun_helper.go (hunk 2 above)
Comment (avi-biton) [reply internal/controller/pipelinerun_helper.go]: Renamed

File: internal/controller/pipelinerun_helper.go [hunk 3]
```diff
@@ -49,9 +49,16 @@ func RemoveFinalizerFromPipelineRun(ctx context.Context, pipelineRun *tektonv1.P
 }
 
 // GetResultsFromPipelineRun extracts results from pipelinerun
+// And add the pipelinerunName
```
Comment (yftacherzog) [inline internal/controller/pipelinerun_helper.go:52]:
```suggestion
// and adds the pipelinerunName
```

**Review by yftacherzog — APPROVED** [review]
lgtm. I think we should create tickets for adding unit tests to this function and others that allow it.

File: internal/controller/pipelinerun_helper.go (hunk 3 above)
Comment (avi-biton) [reply internal/controller/pipelinerun_helper.go]: Done

**Comment by avi-biton** [comment]
> lgtm. I think we should create tickets for adding unit tests to this function and others that allow it.

I am working on it :)

**Comment by avi-biton** [comment]
@yftacherzog PTAL again, I changed the function to return an error in case Results is empty (no results)

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**